test/file.test.js

```diff
@@ -236,7 +236,7 @@ describe('file', function() {
                 assert.ok(elapsed >= 500, `elapsed time: ${elapsed}`);
                 assert.ok(e.message.includes('GET'));
                 assert.ok(e.message.includes('connect failed'));
-                assert.ok(e.message.includes('ENOTFOUND'));
+                assert.ok((e.message.includes('ENOTFOUND') || e.message.includes('EAI_AGAIN'))); 
```

Comment:
I assume this is because different nodejs versions have different error messages

Reply:
Because of the different OS, actually (this one's Ubuntu).